### PR TITLE
Create CalculateX.R Closes #2

### LINKED ISSUE #2: Erstellt ein R Skript

### PR DESCRIPTION
**Pull request by LinaMarieD**
Calculate an x.

### CODE REVIEW

**Comment by LinaMarieD** [comment]
Passt das so?

File: CalculateX.R [hunk 1]
```diff
@@ -0,0 +1 @@
+x = 49-7
```
Comment (grewered) [inline CalculateX.R]: what about y?

**Review by grewered — CHANGES_REQUESTED** [review]

**Comment by grewered** [comment]
looks good now

**Review by grewered — APPROVED** [review]
from here as well

**Review by nickhaf — APPROVED** [review]
Great, thanks

**Pull request merged**